### PR TITLE
Adjust heat assumptions

### PR DESCRIPTION
**Pull request by ClaraBuettner**
This branch adjusts the heat demands and district heating shares in all countries based on hotmaps and other sources. 
The input file for the heat demands is calculated in egon-data (currently on this branch: https://github.com/openego/eGon-data/tree/features/%23278-adjust-heat-assumptions-PyPSA-Eur-Sec). 
@EvaWie implemented it. We both tested it locally (due to less RAM only every 10th hour) and everything was working. 
@EvaWie also implemented a setting in the configuration file to decide if district heating shares for egon-data are used.

### CODE REVIEW

**Comment by EvaWie** [comment]
To avoid confusion, the input file for the heat demands is downloaded in egon-data, as @ClaraBuettner mentioned above, but the calculation is done here in [scripts/adjust_energy_totals.py](https://github.com/openego/pypsa-eur-sec/pull/7/commits/74dc40d5542871633f409040eee327fd7259d221).

File: scripts/adjust_energy_totals.py [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+         Implementation: Load the of the csv file and change the data.
+
+    Notes
+    -----
+        None.
+
+    """
+    # implementation is still missing
+
+    return None
```
Comment (EvaWie) [inline scripts/adjust_energy_totals.py]: The adjust_district_heating_shares() function can be deleted. The adjustment of the district heating shares was implemented in a different way.

File: scripts/adjust_energy_totals.py [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+    'total residential cooking', 'electricity residential cooking'
+    'total residential', 'electricity residential'
+    'total services cooking', electricity services cooking',
+    'total services', 'electricity services'
+    """
+
+    return energy_totals
+
+
+def adjust_CO2_totals():
```
Comment (ulfmueller) [inline scripts/adjust_energy_totals.py]: I would suggest to get rid of this and instead create a ticket with that task or question

File: scripts/adjust_energy_totals.py (hunk 1 above)
Comment (ulfmueller) [reply scripts/adjust_energy_totals.py]: shall I do it @ClaraBuettner ?

**Review by ulfmueller — COMMENTED** [review]
it looks good. I guess we can clean up a bit and then merge... @ClaraBuettner Will you do so?

File: scripts/adjust_energy_totals.py (hunk 2 above)
Comment (ClaraBuettner) [reply scripts/adjust_energy_totals.py]: Sorry, I already deleted the empty functions but didn't push it.

**Review by ulfmueller — APPROVED** [review]

**Comment by ClaraBuettner** [comment]
@EvaWie wants to add more documentation. We can either merge this now and update the documentation on another branch or wait with the merge until it is done. 
Both options are fine for me. @ulfmueller What do you think?

**Comment by ulfmueller** [comment]
I don't care right now...

**Pull request merged**